src/cv2pdb.cpp

```diff
@@ -842,16 +843,30 @@ int CV2PDB::addFieldNestedType(codeview_fieldtype* dfieldtype, int type, const c
 
 int CV2PDB::addFieldEnumerate(codeview_fieldtype* dfieldtype, const char* name, int val)
 {
+	int len = sizeof (dfieldtype->enumerate_v1);
+	BYTE *buffer = (BYTE*)dfieldtype;
+
 	dfieldtype->enumerate_v1.id = v3 ? LF_ENUMERATE_V3 : LF_ENUMERATE_V1;
 	dfieldtype->enumerate_v1.attribute = 0;
-	//assert(val < LF_NUMERIC);
-	dfieldtype->enumerate_v1.value = val;
-	int len = cstrcpy_v(v3, (BYTE*)(&dfieldtype->enumerate_v1 + 1), name);
-	len += sizeof (dfieldtype->enumerate_v1);
 
-	unsigned char* p = (unsigned char*) dfieldtype;
+	// Emit the enumerator value

```

Comment:
better just call write_numeric_leaf() here.

Reply:
Nice, I did not know `write_numeric_leaf` existed. :-) I’ll update this commit.